File: backend/app/schemas/project.py
from pydantic import BaseModel
from datetime import datetime

class ProjectSummary(BaseModel):
    id: str
    name: str
    status: str
    owner: str
    updatedAt: datetime

class ProjectStats(BaseModel):
    total: int
    active: int
    todayCalls: int
    health: float

File: packages/shared-types/src/project.ts
export interface ProjectSummary {
    id: string;
    name: string;
    status: 'active' | 'inactive' | 'pending';
    owner: string;
    updatedAt: string;
}

export interface ProjectStats {
    total: number;
    active: number;
    todayCalls: number;
    health: number; // 0-100
}